test/log_buffer/decorator_test.rb

```diff
@@ -15,6 +16,29 @@ def setup
     @decorator = ::Proxy::LogBuffer::Decorator.new(@logger, @buffer)
   end
 
+  def test_should_pass_logs
+    @logger.expects(:add).with(DEBUG, "message")
+    @decorator.debug("message")
+  end
+
+  def test_should_pass_logs_to_syslog
```

Comment:
I don't see why this test is needed. You verified the interface in test_should_pass_logs (probably worth renaming to test_should_pass_logs_to_logger).